trieste/acquisition/optimizer.py

```diff
@@ -140,3 +143,44 @@ def target_func_with_vectorized_inputs(
         return tf.reshape(vectorized_points, [batch_size, -1])  # [B, D]
 
     return optimizer
+
+
+def batchify_greedy(
+    batch_size_one_optimizer: AcquisitionOptimizer[SP],
+    batch_size: int,
+) -> AcquisitionOptimizer[SP]:
+    """
+    A wrapper around our :const:`AcquisitionOptimizer`s. This class wraps a
```

Comment:
```suggestion
    r"""
    A wrapper around our :const:`AcquisitionOptimizer`\ s. This class wraps a
```
you might wanna watch out for remaining PR comments - this was in the last PR but wasn't fixed. You'll need to fix it for `batchify` too - the docstring renders all wrong there